tests/phpunit/tests/option/determineOptionAutoloadValue.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @group option
+ */
+class Tests_Determine_Option_Autoload_Value extends WP_UnitTestCase {
+	public function set_up() {
+		add_filter( 'max_option_size', array( $this, 'filter_max_option_size' ) );
+		parent::set_up();
+	}
+
+	/**
+	 * @covers ::determine_option_autoload_value
+	 *
+	 * @ticket 42441
+	 */
+	public function test_small_option() {
+		$test = determine_option_autoload_value( 'foo', 'bar', null );
+		$this->assertSame( 'default-yes', $test );
+	}
+
+	/**
+	 * @covers ::determine_option_autoload_value
+	 *
+	 * @ticket 42441
+	 */
+	public function test_large_option() {
+		$value = file( DIR_TESTDATA . '/formatting/entities.txt' );
+		$test  = determine_option_autoload_value( 'foo', $value, null );
+		$this->assertSame( 'default-no', $test );
+	}
+
+	/**
+	 * @covers ::determine_option_autoload_value
+	 *
+	 * @ticket 42441
+	 */
```

Comment:
Please move the `@covers` annotations below the `@ticket` as it is the standard order. Or even better: Since this entire test class is just for the one function, let's remove the `@covers` annotations from the methods and instead put it on the overall class.
```suggestion
	/**
	 * @ticket 42441
	 */
	public function test_small_option() {
		$test = determine_option_autoload_value( 'foo', 'bar', null );
		$this->assertSame( 'default-yes', $test );
	}

	/**
	 * @ticket 42441
	 */
	public function test_large_option() {
		$value = file( DIR_TESTDATA . '/formatting/entities.txt' );
		$test  = determine_option_autoload_value( 'foo', $value, null );
		$this->assertSame( 'default-no', $test );
	}

	/**
	 * @ticket 42441
	 */
```